src/components/donation/Steps/DonateMethods/PayPal/Checkout.tsx

```diff
@@ -51,14 +51,14 @@ export default function Checkout({ orderId, onBack }: Props) {
                 }).unwrap();
 
                 if ("details" in order) {
-                  if (order.details.at(0)?.issue === "INSTRUMENT_DECLINED") {
+                  if (order.details[0]?.issue === "INSTRUMENT_DECLINED") {
```

Comment:
Since `order.details` array is now defined as always having a length of 1 (see [src/types/aws/apes/paypal.ts](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2663/files#diff-0aeab8fdf7805cf0b7c7e68781f1f4041c46f9cfe025e1c87799f2c0b6d2bf5fR21)), changing the access to square bracket format adds type safety (changing `0` to `1` will cause a compiler error)